Source/GitReleaseManager/Configuration/ConfigurationProvider.cs

```diff
@@ -75,5 +79,18 @@ private static string GetConfigFilePath(string targetDirectory)
         {
             return Path.Combine(targetDirectory, "GitReleaseManager.yaml");
         }
+
+        private static void EnsureDefaultConfig(Config configuration)
+        {
+            if(configuration.Create.ShaSectionHeading == null)
+            {
+                configuration.Create.ShaSectionHeading = "__SHA256 Hashes of the release artifacts__";
+            }
+
+            if(configuration.Create.ShaSectionLineFormat == null)
+            {
+                configuration.Create.ShaSectionLineFormat = "`{1}\t{0}`";

```

Comment:
Same as my question above:

Should this be in a list?
IE:
```suggestion
                configuration.Create.ShaSectionLineFormat = "- `{1}\t{0}`";
```

Reply:
Yes, I think you are right.  I will fix this up.